libraries/platforms/bigblade-verilator/compilation.mk

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2019, University of Washington All rights reserved.
+# 
```

Comment:
Is this the same file as in bigblade-vcs?

Reply:
It is different because we don't generate main.so, which I understand isn't necessary for verilator

Reply:
Yeah, fair